source/dotnet/Library/CppConsoleApp/CppConsoleApp.vcxproj

```diff
@@ -0,0 +1,137 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
even though this binary doesn't ship, it'll probably save a fair amount of pain if you turn on compliance flags

https://github.com/microsoft/AdaptiveCards/blob/d3c25221fdebaa8631e96e132e5329b71e5ccab1/source/uwp/winui3/AdaptiveCardsObjectModel/AdaptiveCardsObjectModel.vcxproj#L134-L136

https://github.com/microsoft/AdaptiveCards/blob/d3c25221fdebaa8631e96e132e5329b71e5ccab1/source/uwp/winui3/AdaptiveCardsObjectModel/AdaptiveCardsObjectModel.vcxproj#L107-L108

(I'm probably missing some here)